### DISCUSSION TITLE
How to choose graphics card serving for WSLg

### DISCUSSION THREAD

**Discussion by seflerZ** [microsoft/wslg#780] [Q&A] [answered]
I've 3 graphics cards in my system and the default chosen was Nvidia GTX 1050. How can I switch to another one which is Intel UHD 770?

**Comment by seflerZ** [comment, marked as answer]
Well, after digging for some documentations, I found [this](https://gitlab.freedesktop.org/mesa/mesa/blob/main/docs/drivers/d3d12.rst) which means an environment variable will work, like:
```
export MESA_D3D12_DEFAULT_ADAPTER_NAME="Intel(R) UHD Graphics 770"
```

**Reply by limingchina** [reply]
Works for me as well when I want to choose my NVidia card: "NVIDIA Quadro M1200"
